config/database.yml

```diff
@@ -1,25 +1,16 @@
-# SQLite version 3.x
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
-#
 default: &default
   adapter: postgresql
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   timeout: 5000
   username: <%= ENV['DB_USERNAME'] %>
   password: <%= ENV['DB_PASSWORD'] %>
-  host: <%= ENV['DB_HOSTNAME'] || 'localhost' %>
-  port: <%= ENV['DB_PORT'] || '5432' %>
+  host: <%= ENV['DB_HOSTNAME'] %>
+  port: <%= ENV['DB_PORT'] %>
 
 development:
   <<: *default
-  database: <%= ENV['DB_DATABASE'] || 'bat_apply_dev' %>
```

Comment:
The deployment pipeline requires the DB_DATABASE, so removing this, as proposed, will break stages further down the pipeline. What is the reasoning behind removing the override functionality?